src/main/java/bookbob/entity/Patient.java

```diff
@@ -119,18 +121,28 @@ public void setMedicalHistories(ArrayList<String> medicalHistories) {
 
     @Override
     public String toString() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        String dateOfBirthString = "";
+        if(getDateOfBirth() != null) {
+            dateOfBirthString = getDateOfBirth().format(formatter);
+        }
         return "Name: " + getName() + ", NRIC: " + getNric() +
                 ", Phone: " + getPhoneNumber() +  ", Address: " + getHomeAddress() +
-                ", DOB: " + getDateOfBirth() + ", Allergy: " + getAllergies() +
+                ", DOB: " + dateOfBirthString + ", Allergy: " + getAllergies() +
                 ", Sex: " + getSex() + ", Medical History: " + getMedicalHistories();
     }
 
     //@@author PrinceCatt
     @Override
     public String convertPatientToOutputText() {
         String output = "";
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+        String dateOfBirthString = "";
+        if(getDateOfBirth() != null) {
```

Comment:
Perhaps can add a space after the word "if" for better readability? I noticed some other instances too